extensions/amp-video/0.1/amp-video.js

```diff
@@ -213,9 +213,7 @@ class AmpVideo extends AMP.BaseElement {
 
     this.video_ = element.ownerDocument.createElement('video');
     // Manage video if the sources contain bitrate or amp-orig-src will be expanded to multiple bitrates.
-    if (
-      this.element.querySelector('source[data-bitrate], source[amp-orig-src]')
-    ) {
+    if (this.hasAnyCachedSources_() || this.hasAnyBitrateSources_()) {
```

Comment:
Is this change needed?

Reply:
It's more accurate since the `hasAnyCahchedSources` method contains more than just a querySelector